### PR TITLE
Feat/navbar

### PR DESCRIPTION
**Pull request by SupriyaShekar123**
# What does this pull request do ?

In this pull request, new logo is added and button is linked to external booking link. Some changes are made in styling (backgroundcolor,padding,height,width).

### CODE REVIEW

File: src/Components/NavBar/index.css [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-  /* line-height: 5; */
-  padding-top: 5px;
-  font-weight: normal;
-  color: #2b3a44;
-  text-decoration: none;
-
-  /* font-family: "Nunito Sans", sans-serif; */
+@import url("https://fonts.googleapis.com/css?family=Antic+Didone%7CNunito+Sans:400,700&amp;display=swap");
+.navigation_bar>a {
+    /* padding-top: 5px; */
```
Comment (jendhordejan) [inline src/Components/NavBar/index.css:3]: please delete if not needed

File: src/Components/NavBar/index.css [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+    font-weight: normal;
+    color: #2b3a44;
+    text-decoration: none;
+    font-family: "Antic Didone", "system-ui", "Segoe UI", "Roboto", "Helvetica", "Arial", sans-serif;
+    text-transform: capitalize;
+    font-size: 1.25rem;
+    line-height: 1rem;
+    /* margin-bottom: 1.5rem; */
```
Comment (jendhordejan) [inline src/Components/NavBar/index.css:11]: please delete if not needed

File: src/Components/NavBar/index.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default function index() {
       {"   "}
```
Comment (jendhordejan) [inline src/Components/NavBar/index.js:20]: can we remove this?

**Review by jendhordejan — CHANGES_REQUESTED** [review]

File: src/Components/NavBar/index.css (hunk 2 above)
Comment (mitraws) [reply src/Components/NavBar/index.css:11]: Will do during refactor.

File: src/Components/NavBar/index.js (hunk 3 above)
Comment (mitraws) [reply src/Components/NavBar/index.js:20]: Will do during refactor!

**Review by mitraws — APPROVED** [review]
Thx!

**Pull request merged**